FEAT: add CheckSwitchLanguage in Test.cs, create LanguagesPage.cs

khayitmuradov opened the pull request:
created LanguagesPage.cs in Pages folder
added CheckSwitchLanguage() method in Test.cs

khayitmuradov committed:
FEAT: add CheckSwitchLanguage in Test.cs, created LanguagesPage.cs

NikitaGavruk commented on UI/Pages/LanguagesPage.cs:
```diff
@@ -0,0 +1,44 @@
+        private static readonly By uzbekLanguagePage = By.LinkText("Uzbek Wikipedia");
+        private static readonly By uzbekWikiPedia = By.LinkText("uz.wikipedia.org");
+
+        public void ClickPopUpWindowButton()
```
Make it please via fluent page object pattern

NikitaGavruk commented on UI/Tests/Test.cs:
```diff
@@ -73,5 +73,23 @@ public void CheckNotLoggedWarningOnEditPage()
+            Assert.That(Driver.Title, Is.EqualTo("Языковые разделы Википедии — Википедия"));
+            languagesPage.ClickEnglishLanguageButton();
+            Assert.That(Driver.Title, Is.EqualTo("List of Wikipedias - Wikipedia"));
+            languagesPage.ClickUzbekLanguageButton();
```
Add please log info here too as you did it on 85 line

NikitaGavruk commented on UI/Tests/Test.cs:
```diff
@@ -73,5 +73,23 @@ public void CheckNotLoggedWarningOnEditPage()
+            languagesPage.ClickEnglishLanguageButton();
+            Assert.That(Driver.Title, Is.EqualTo("List of Wikipedias - Wikipedia"));
+            languagesPage.ClickUzbekLanguageButton();
+            languagesPage.ClickUzbekWikipediaButton();
```
Why you did not verify Uzbek language?

NikitaGavruk commented:
What does it mean "FEAT" at name of this PR?

khayitmuradov commented:
FEAT means feature no?  I read it from that documentation about commiting which you gave

khayitmuradov committed:
remove LanguagesPage
create EnglishLanguagePage
create ListOfWikipediasPage
create RussianLanguagePage
create UzbekLanguagePage
create LanguagePagesSteps
add CheckSwitchLanguages method in Test.cs

khayitmuradov commented:
> What does it mean "FEAT" at name of this PR?
Reworked Languages Part: #21

NikitaGavruk commented:
:) I did not read it. Anyway we do not need it. We just need to have clear commits name
